src/Command/CleanupCommand.php

```diff
@@ -15,14 +15,12 @@
 
 class CleanupCommand extends Command
 {
-    /** @var string */
+    /**
+     * @inheritdoc
+     */
```

Comment:
Nevim jestli jsou nejake preference nebo, da se klidne zapsat i jen kratkym

```suggestion
    /** @inheritdoc */
```

Reply:
jo why not, nepřemýšlel jsem nad tím :)